src/main/resources/application.yaml

```diff
@@ -97,4 +97,4 @@ rhsm-subscriptions:
       tasks:
         topic: ${OPENSHIFT_METERING_TASK_TOPIC:platform.rhsm-subscriptions.openshift-metering-tasks}
         kafka-group-id: ${OPENSHIFT_METERING_TASK_GROUP_ID:openshift-metering-task-processor}
-
+  prometheus-latency-duration: ${PROMETHEUS_LATENCY_DURATION:PT4H}
```

Comment:
I prefer the "more readable" format the Spring Boot supports (https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-conversion-duration) just "4h". I had to google PT4H, so regardless, I learned something :-)

Reply:
:+1:

Reply:
Could have used this in the metrics PR. :)

Reply:
Well would you look at that....didn't know there even was a more readable version.  Just a heads up on this though...while spring magic makes this work a property...junit doesn't have the same magic for the @ParameterizedTest I made.

```
Error converting parameter at index 1: Failed to convert String "0h" to type java.time.Duration
org.junit.jupiter.api.extension.ParameterResolutionException: Error converting parameter at index 1: Failed to convert String "0h" to type java.time.Duration
	at org.junit.jupiter.params.ParameterizedTestMethodContext.parameterResolutionException(ParameterizedTestMethodContext.java:273)
```